### PR TITLE
Prep amazon.aws 7.0.0 release

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Prep amazon.aws 7.0.0 release. Release date is on November 2.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

several

### CODE REVIEW

**Comment by gravesm** [comment]
Shouldn't this be merging against a stable-7.0 branch instead of main?

**Comment by alinabuzachis** [comment]
> Shouldn't this be merging against a stable-7.0 branch instead of main?

We can merge against main and then create a stable-7 branch from there.  If am not mistaken, this is what we've done for the other releases. @tremble

**Comment by tremble** [comment]
community.aws merged into stable-6 first https://github.com/ansible-collections/community.aws/pull/1813

**Review by hakbailey — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -2422,3 +2422,151 @@ releases:
...
+        ``boto3<1.26.0``. Most modules will continue to work with older versions of
+        the AWS SDK, however compatability with older versions of the SDK is not guaranteed
+        and will not be tested. When using older versions of the SDK a warning will
+        be emitted by Ansible (https://github.com/ansible-collections/amazon.aws/pull/1763).
+      - amazon.aws collection - due to the AWS SDKs announcing the end of support
+        for Python less than 3.7 (https://aws.amazon.com/blogs/developer/python-support-policy-updates-for-aws-sdks-and-tools/)
+        support for Python less than 3.7 by this collection wss been deprecated in
+        release 6.0.0 and removed in release 7.0.0. (https://github.com/ansible-collections/amazon.aws/pull/1763).
+      - module_utils - ``module_utils.urls`` was previously deprecated and has been
+        removed ().
```
Comment (tremble) [inline changelogs/changelog.yaml:2438]:
```suggestion
        removed (https://github.com/ansible-collections/amazon.aws/pull/1540).
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,78 @@ amazon.aws Release Notes
...
+- s3_object - Allow recursive copy of objects in S3 bucket (https://github.com/ansible-collections/amazon.aws/issues/1379).
+- s3_object - use fstrings where appropriate (https://github.com/ansible-collections/amazon.aws/pull/1802).
+
+Breaking Changes / Porting Guide
+--------------------------------
+
+- The amazon.aws collection has dropped support for ``botocore<1.29.0`` and ``boto3<1.26.0``. Most modules will continue to work with older versions of the AWS SDK, however compatability with older versions of the SDK is not guaranteed and will not be tested. When using older versions of the SDK a warning will be emitted by Ansible (https://github.com/ansible-collections/amazon.aws/pull/1763).
+- amazon.aws collection - due to the AWS SDKs announcing the end of support for Python less than 3.7 (https://aws.amazon.com/blogs/developer/python-support-policy-updates-for-aws-sdks-and-tools/) support for Python less than 3.7 by this collection wss been deprecated in release 6.0.0 and removed in release 7.0.0. (https://github.com/ansible-collections/amazon.aws/pull/1763).
+- module_utils - ``module_utils.urls`` was previously deprecated and has been removed ().
```
Comment (tremble) [inline CHANGELOG.rst:55]:
```suggestion
- module_utils - ``module_utils.urls`` was previously deprecated and has been removed (https://github.com/ansible-collections/amazon.aws/pull/1540).
```

**Review by tremble — CHANGES_REQUESTED** [review]

**Pull request closed without merging**